radicle-cli/examples/rad-node.md

```diff
@@ -0,0 +1,99 @@
+The radicle node is our daemon friend that is running as a background
+process. It allows us to interact with the network as well as storing
+some key data that we may be interested in.
+
+<details>
+<summary>🚧 `rad node start` is under construction 🚧</summary>
+If the node is not running we can start it by using the `rad node
+start` command:
+
+<!-- ``` -->
+<!-- $ rad node start -->
+<!-- ``` -->
+</details>
+
+We can confirm the status of the node at any time by using the `rad
+node status` command (or just `rad node` for short):
+
+```
+$ rad node status
+✓ The node is running
```

Comment:
Would be cool to show the listen address here if it's easy; as well as the node socket file. Otherwise maybe in another PR.

Reply:
I'd say let's do that in a follow-up since status is just using `is_running`, but maybe we'd want something more like a `status` method in `Handle` which has more verbose info.